cmd/kube-controller-manager/app/names/controller_names.go

```diff
@@ -0,0 +1,108 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package names
+
+import cpnames "k8s.io/cloud-provider/app/names"
+
+// canonical controller names
```

Comment:
can this explain where these names are used (command-line invocations, events, contextual logging...?), and the policy around changing them (I think the policy will be "don't")

Reply:
good idea, I have added a documentation for these names inside both of the names packages